src/cephadm/cephadm

```diff
@@ -152,6 +152,169 @@ class Monitoring(object):
         },
     }  # type: ignore
 
+##################################
+
+class NFSGanesha(object):
+    """Defines a NFS-Ganesha container"""
+
+    components = {
+        'nfs' : {
+            'config-json-files' : [
+                'ganesha.conf',
+            ],
+        },
+    }
+
+    entrypoint = '/usr/bin/ganesha.nfsd'
+    daemon_args = ['-F', '-L', 'STDERR']
+
+    port_map = {
+        "nfs" : 2049,
+    }
```

Comment:
```suggestion
   port = 2049
```
(just wondering why we need a map, as we already are in the class `NFSGanesha`. Or, which other keys do you expect in this map?)

Reply:
Just trying to maintain consistency with the existing `Monitoring` class so that it's easy to refactor if we find an opportunity to consolidate and/or reuse some of this logic ...